### DISCUSSION TITLE
Change alt + w keybind

### DISCUSSION THREAD

**Discussion by kennjoma** [kangyu-california/PersistentWindows#316] [Q&A] [answered]
Is there a way to change alt + w keybind...or even just turn that keybind off?

**Comment by kangyu-california** [comment, marked as answer]
> Is there a way to change alt + w keybind...or even just turn that keybind off?

-webpage_commander_window=0
      Turn off Alt+W hotkey for webpage commander window

-hotkey "Q"
   register Alt + Q as the hotkey to (de)activate webpage commander window, default hotkey is "W" (Alt + W)

**Reply by kennjoma** [reply]
Appreciate the reply. Where am I putting this though?

**Reply by kangyu-california** [reply]
#313
